Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3112,6 +3168,39 @@ public void run() {
 		});
 	}
 
+	private void processRedeemRewardsResponse(final ServerResponse resp) {
+		boolean updateListener = false;
```

Comment:
If you're adding something new, you might as well make it clean. No use continuing bad habits.